From b5c3a610c367df593433beb417542cc76cd14a49 Mon Sep 17 00:00:00 2001
From: hienhuynh <hien.huynh.px@renesas.com>
Date: Tue, 8 Mar 2022 15:11:44 +0700
Subject: [PATCH] Update correct base number of VTOP ioctl

mmngr kernel module changed VTOP ioctl base number to 9.
So this commit is created to update ect base number of VTOP ioctl.

Signed-off-by: hienhuynh <hien.huynh.px@renesas.com>
---
 gstvspmfilter.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gstvspmfilter.h b/gstvspmfilter.h
index 601a7af..55f0090 100644
--- a/gstvspmfilter.h
+++ b/gstvspmfilter.h
@@ -76,7 +76,7 @@ struct MM_PARAM {
 
 /* mmngr private define */
 #define MM_IOC_MAGIC 'm'
-#define MM_IOC_VTOP	_IOWR(MM_IOC_MAGIC, 7, struct MM_PARAM) 
+#define MM_IOC_VTOP	_IOWR(MM_IOC_MAGIC, 9, struct MM_PARAM) 
 
 typedef struct _GstVspmFilter GstVspmFilter;
 typedef struct _GstVspmFilterClass GstVspmFilterClass;
-- 
2.25.1

